experimental/QuadFormAndIsom/src/enumeration.jl

```diff
@@ -1111,67 +1844,438 @@ end
 
 ###############################################################################
 #
-#  Testing functions
+#  Helper for managing conditions on isometries to enumerate
```

Comment:
New section about handling conditions on eigenlattices and minimal/characteristic polynomials (as well as understanding the restrictions on the powers of the isometries)